deltachat-ios/AppDelegate.swift

```diff
@@ -48,7 +50,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // `didFinishLaunchingWithOptions` is _not_ called
     // when the app wakes up from "suspended" state
     // (app is in memory in the background but no code is executed, IO stopped)
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication, willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
the link was wrong of the post that I deleted, but I still think from reading docs the launch options should be read from `didFinishLaunchingWithOptions`

Reply:
yip, i just want to do some experiments.

but i am still wondering if it is correct that we start the ui unconditionally, so, also on background fetch. or i misunderstood the code :)